app/Services/ArticleService.php

```diff
@@ -37,6 +37,10 @@ public function getCategorizedArticlesByMagazine($magazineId)
         // For now, simply sorting categories alphabetically.
         ksort($categories);
 
+        foreach ($categories as $index => $category) {
+            $categories[$index] = collect($category)->sortByDesc('created_at');
+        }
```

Comment:
Sure. I am on leave right now and having limited Internet connectivity through mobile.

I will be online on slack at around 4:30 pm IST. Let's chat at that time. You can ping me any time. I have slack access on my tablet but can't have extended chat on tablet.